.env

```diff
@@ -1,2 +1,3 @@
 SKIP_PREFLIGHT_CHECK=true
 REACT_APP_VERSION=$npm_package_version
+REACT_APP_GOOGLE_MAP_KEY=AIzaSyArM4e0n53tWyK5drjXP03OmovvVJHk8OU
```

Comment:
@mvines is this a problem? I remember this needed to be invalidated before

cc: @manuel-calavera

Reply:
this key is fine. same as here https://github.com/solana-labs/networkexplorer/blob/d426a5086cc1f2c10999419bf84e76ccfdebbb3d/src/v1/BxDialogWorldMap.jsx#L19

Reply:
it has special domain restriction, so must be safe.